### DISCUSSION TITLE
Ideas for 7th Annual International Humanitarian Mapathon 2024

### DISCUSSION THREAD

**Discussion by hfu** [UNopenGIS/7#417] [Ideas]
As discussed in https://github.com/UNopenGIS/7/issues/385, let's think about a session with the process below: 

1. We use FSQ Studio as a "next generation GIS".
2. We use some simple data from OCHA HDX.
3. We develop an example on FSQ Studio and create a document on GitHub Discussion <strike>HackMD</strike>.

I decided to use GitHub discussion because it can capture screenshots easier than HackMD.

**Comment by hfu** [comment]
## Access FSQ Studio
Access https://studio.foursquare.com/home. You probably need to sing up first. After that, you will see the menu as below. 

<img width="640" alt="image" src="https://github.com/UNopenGIS/7/assets/18297/5215b065-4e11-4195-80d7-a8b53ce44822">

**Comment by hfu** [comment]
## Choose your data from OCHA HDX
https://data.humdata.org/ is the Humanitarian Data Exchange (HDX). 

<img width="1007" alt="image" src="https://github.com/UNopenGIS/7/assets/18297/a65517e9-ddaf-4448-852e-a83250d06457">

**Comment by hfu** [comment]
## Download the data - "Ukraine: Border Crossings"
https://data.humdata.org/dataset/ukraine-border-crossings

Let's download "UKR_Border_Crossings_20230530.geojsonGeoJSON (44.5K)"

<img width="853" alt="image" src="https://github.com/UNopenGIS/7/assets/18297/09a0461f-dd36-4be0-bde1-598e1beee24d">

**Comment by hfu** [comment]
## Create a map in FSQ Studio and upload the GeoJSON data

<img width="985" alt="image" src="https://github.com/UNopenGIS/7/assets/18297/61458a1c-db4e-4634-989a-10a3d9e7d22d">

<img width="940" alt="image" src="https://github.com/UNopenGIS/7/assets/18297/38909b9a-8345-448d-9f19-d422955c37f9">

<img width="1506" alt="image" src="https://github.com/UNopenGIS/7/assets/18297/be4452a7-11b7-41df-92ec-90ae9ae01dac">

<img width="284" alt="image" src="https://github.com/UNopenGIS/7/assets/18297/3252f2b6-e3bf-4e27-b2dd-a629e1db2da0">

<img width="766" alt="image" src="https://github.com/UNopenGIS/7/assets/18297/4db49fc6-8f2b-4e7b-afcf-fc229477397e">

**Comment by hfu** [comment]
### Explore the details!

<img width="923" alt="image" src="https://github.com/UNopenGIS/7/assets/18297/9fbedfdf-3f39-4d09-aaa5-4aa3bcd6e11e">

<img width="908" alt="image" src="https://github.com/UNopenGIS/7/assets/18297/d1edfe3e-f2be-49ea-b656-0fb1803d21ac">

**Comment by hfu** [comment]
## Publish!
I have published the map at https://studio.foursquare.com/public/e583ec66-08b4-4d3b-912c-0bd016ca47cc.

**Comment by yohman** [comment]
@hfu Oh wow, this is very promising! I have never used FSQ but it looks exactly the same as kepler, so I assume it is using the same platform. I worry that it is not open source, but maybe we cannot avoid this. 

As for the data, I think we should find and curate a data package for the participants. So if we use HDX, maybe the topic location is "Ukraine" and we give the the link to the Ukraine Grid: https://data.humdata.org/group/ukr. However, aside from the border crossing data, I don't know if there are other datasets that participants can use to create maps?  Unlike a workshop where everybody is taught to create the same map, we want users to be able to create different kinds of maps. For example, this output from them is really nice:

<img width="1060" alt="image" src="https://github.com/UNopenGIS/7/assets/825990/ebd45c0d-0634-46e7-9d46-ffc9edf6a12a">

But I don't know if the Ukraine grid provides the data to be able to produce something similar? If we only provide border crossings, then I think most people will create maps that are too similar to one another, and it will be hard for you to judge. So maybe adding some layers such as:

- in/out migration data
- administrative boundaries that already include demographic statistics (population, poverty, etc)
- incident data by date (conflict zones, etc)
- building data (demolished, destroyed, etc)
- road data (unpassable, restricted, etc)
- other?

These are just some thoughts, but having just a few of these would make it a more complete data package. Then we would have more variety on what participants choose to submit. We are close!!

**Reply by hfu** [reply]
Thank you @yohman. We proceed as proposed.

**Comment by hfu** [comment]
# Data Package issue: where is the project site for the event?

## Central African Republic (CAR)? 
- https://peacekeeping.un.org/en/mission/minusca
- https://minusca.unmissions.org/
- https://www.un.org/geospatial/content/minusca-deployment-october-2023
- https://reports.unocha.org/en/country/car/
- https://www.unocha.org/publications/report/central-african-republic/central-african-republic-situation-report-6-mar-2024

**Reply by yohman** [reply]
The CAR report from UNOCHA is very compelling. As an option, we can use the mapathon to educate students on the situation, which most would not be aware of, or have ever seen a map of the region (I barely have). So we could provide a data package that includes cities, airports, admin boundaries, census data, as well as country data for the surrounding regions (Chad, Sudan, Congo, etc).

**Comment by hfu** [comment]
# `Global Observer`: a minimum web map app
I created `Global Observer`, a minimum web map app implemented on Observable. 

https://observablehq.com/d/652ef8ca26776146

<img width="1298" alt="image" src="https://github.com/UNopenGIS/7/assets/18297/730a4502-cda3-4fcc-8663-8e198cfb73d9">

**Comment by hfu** [comment]
# Data Package for Central African Republic (CAR)
Let's develop a Data Package for Central African Republic. 

## Study Area
Here is a draft extent of the study area. If we can extract data from global data set, let's use the GeoJSON extent below: 

```geojson
{
  "type": "Feature",
  "properties": {},
  "geometry": {
    "type": "Polygon",
    "coordinates": [
      [
        [
          14.095997386323063,
          11.289116216228479
        ],
        [
          27.852508628279054,
          11.289116216228479
        ],
        [
          27.852508628279054,
          1.6596763896651936
        ],
        [
          14.095997386323063,
          1.6596763896651936
        ],
        [
          14.095997386323063,
          11.289116216228479
        ]
      ]
    ]
  }
}
```

## Source
Let's create a list of the source and decide who does what. 

- [World Population Density Map](https://www.kontur.io/portfolio/population-dataset/) (@hfu) 
- TODO

**Comment by hfu** [comment]
We are working on other data. So, we can close this discussion.
